### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by spetrychenko**
Increase code coverage up to 80% for xs2a-adapter-service-remote

### CODE REVIEW

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java [hunk 1]
```diff
@@ -56,13 +56,6 @@ public RemoteAccountInformationService(AccountInformationClient client) {
         objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
     }
 
-    public RemoteAccountInformationService(
```
Comment (artemsnisarenko) [inline xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java:59]: Why are you removing this constructor? This reverts a080d30

File: xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteOauth2ServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    @Test
+    void getAuthorizationRequestUriWithException() throws IOException {
+
+        doReturn(buildHref("blablabla_unexisting_uri@2#^"))
+            .when(oauth2Client)
+            .getAuthorizationUrl(anyMap(), any());
+
+        assertThrows(IOException.class, () -> service.getAuthorizationRequestUri(new HashMap<>(),
```
Comment (artemsnisarenko) [inline xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteOauth2ServiceTest.java:46]: violates https://rules.sonarsource.com/java/RSPEC-5778

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java (hunk 1 above)
Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java:59]: not found usages of it, will revert back

Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java:59]: reverted back

File: xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteOauth2ServiceTest.java (hunk 2 above)
Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteOauth2ServiceTest.java]: adjusted, thanks

Comment (artemsnisarenko) [reply xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteOauth2ServiceTest.java]: please remove curly braces in the lambda, it complains about that too

Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteOauth2ServiceTest.java]: removed

**Pull request merged**